frontend/src/pages/org/browser-profiles-list.ts

```diff
@@ -54,133 +55,128 @@ export class BrowserProfilesList extends LiteElement {
+        >
+          <a
+            class="flex items-center gap-3"
+            href=${`${this.orgBasePath}/browser-profiles/profile/${data.id}`}
+            @click=${this.navLink}
+          >
+            ${data.name}
+          </a>
+          <div class="text-xs text-neutral-500 w-full">
+            <div class="truncate">
+              ${data.description} ${data.description} ${data.description}
+              ${data.description} ${data.description} ${data.description}
               ${data.description}
             </div>
           </div>
-          <div class="col-span-8 md:col-span-1 p-2 text-sm" role="cell">
-            ${new Date(data.created).toLocaleDateString()}
-          </div>
-          <div class="col-span-7 md:col-span-3 p-2 text-sm" role="cell">
-            ${data.origins.join(", ")}
-          </div>
-          <div class="col-span-1 md:col-span-1 flex items-center justify-end">
-            ${this.renderMenu(data)}
-          </div>
-        </div>
-      </a>
+        </btrix-table-cell>
+        <btrix-table-cell class="whitespace-nowrap">
+          <sl-format-date
+            date=${`${data.created}Z`}
+            month="2-digit"
+            day="2-digit"
+            year="2-digit"
+            hour="2-digit"
+            minute="2-digit"
+          ></sl-format-date>
+        </btrix-table-cell>
+        <btrix-table-cell>${data.origins.join(", ")}</btrix-table-cell>
+        <btrix-table-cell class="p-0"
+          >${this.renderActions(data)}</btrix-table-cell
+        >
+      </btrix-table-row>
     `;
-  }
+  };
 
-  private renderMenu(data: Profile) {
+  private renderActions(data: Profile) {
     return html`
-      <sl-dropdown hoist @click=${(e: Event) => e.preventDefault()}>
-        <sl-icon-button
-          slot="trigger"
-          name="three-dots"
-          label=${msg("Actions")}
-          style="font-size: 1rem"
-        ></sl-icon-button>
-        <ul
-          class="text-sm text-neutral-800 bg-white whitespace-nowrap"
-          role="menu"
-        >
-          <li
-            class="p-2 hover:bg-zinc-100 cursor-pointer"
-            role="menuitem"
+      <btrix-overflow-dropdown @click=${(e: Event) => e.preventDefault()}>
+        <sl-menu>
+          <sl-menu-item
             @click=${(e: any) => {
               this.duplicateProfile(data);
-              e.target.closest("sl-dropdown").hide();
             }}
           >
-            <sl-icon
-              class="inline-block align-middle px-1"
-              name="files"
-            ></sl-icon>
-            <span class="inline-block align-middle pr-2"
-              >${msg("Duplicate profile")}</span
-            >
-          </li>
-          <li
-            class="p-2 text-danger hover:bg-danger hover:text-white cursor-pointer"
-            role="menuitem"
+            <sl-icon slot="prefix" name="files"></sl-icon>
+            ${msg("Duplicate profile")}
+          </sl-menu-item>
+          <sl-menu-item
```

Comment:
Should be red to match our other delete buttons, not adding a suggestion because IMO it would be cleaner to put this on a new line and apparently that's _impossible_ for GitHub to do ;)

`style="--sl-color-neutral-700: var(--danger)"`